neo/Consensus/ConsensusService.cs

```diff
@@ -363,6 +345,28 @@ private void OnRecoveryMessageReceived(ConsensusPayload payload, RecoveryMessage
             }
         }
 
+        private void OnRecoveryRequestReceived(ConsensusPayload payload)
+        {
+            if (context.WatchOnly()) return;
+            if (!context.CommitSent())
+            {
+                bool shouldSendRecovery = false;
+                // Limit recovery to be sent from, at least, `f` nodes when the request is from a lower view number.
+                int allowedRecoveryNodeCount = context.F();
+                for (int i = 1; i <= allowedRecoveryNodeCount; i++)
```

Comment:
@erikzhang, I think that we should still use `message.NewViewNumber`, which inserts a round-robin on those who will recover.

Reply:
I will Resolve this because we are discussing the below.